packages/api/src/index.ts

```diff
@@ -156,7 +156,6 @@ class Sdk {
        */
       auth: (...values: string[] | number[]) => {
         core.setAuth(...values);
-        return new Proxy(sdk, sdkProxy);
```

Comment:
This is the part that actually removes the chaining.

Reply:
Can you update the README? There's a handful of references in there talking about how `.auth()` is chainable. You'll need to update the root README and the one in `packages/api` too but I believe the root one is just a copy of the package one.